web/src/components/InvestmentCard.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import { Button, CardHeader, Chip, Grid, makeStyles, TextField, Typography } from '@material-ui/core';
+import { green, yellow } from '@material-ui/core/colors';
+import theme from '../theme';
+import SpinnerInput from './SpinnerInput';
+import CurrencyTextField from '@unicef/material-ui-currency-textfield';
+
+const useStyles = makeStyles({
+  root: {
+    '& [type=number]': {
+      '-moz-appearance': 'textfield',
+      textAlign: 'right'
+    },
+    '& ::-webkit-outer-spin-button': {
+      '-webkit-appearance': 'none',
+      margin: 0,
+    },
+    '& ::-webkit-inner-spin-button': {
+      '-webkit-appearance': 'none',
+      margin: 0,
+    },
+  }
+});
+
+const getPhaseChipColor = (phase) => {
+  switch (phase) {
+    case 'active':
+      return green[400]
+    case 'funding':
+      return yellow[900]
+    case 'completed':
+      return theme.palette.primary.main
+  }
+}
+
+const investmentCardHeader = (phase, loanName) =>
+  <CardHeader
+    style={{backgroundColor: theme.palette.secondary.main, color: theme.palette.primary.contrastText}}
```

Comment:
You can use 'styled' from '@material-ui/core/styles' for this, instead of inline styling

Reply:
Done